app/__init__.py

```diff
@@ -9,7 +9,7 @@
 app = Flask(__name__)
 
 # Configurations
-app.config.from_object('config')
+# app.config.from_object('config')
```

Comment:
Why are we not loading the configurations?

Reply:
The above line in any case was just a placeholder. There is no config file in the repo right now. The import_string seems to be throwing an exception rather than just ignoring.